### ISSUE TITLE
WFS Layer "LightUnit" seems to be broken

### ISSUE THREAD

**Issue by JakobMiksch** [compassinformatics/cpsi-mapview#183]
This WFS Layer does not display in CPSI-Mapview. In QGIS or with OGR it does not open either. The WMS from the same URL however works for me.

```
    {
      "layerType": "wfs",
      "text": "Light Unit WFS",
      "layerKey": "LIGHT_UNIT_WFS",
      "url": "https://plmonaghandev.compass.ie/mapserver/?",
      "geometryProperty": "msGeometry",
      "featureType": "LightUnit",
      "sldUrl": "resources/data/styling/LightUnit_Unit_Type.xml",
      "openLayers": {
        "visibility": false
      }
    },
```

**Comment by geographika** [comment]
@JakobMiksch - we're in the process of setting up stable services just for this project in #173
I've tried to fix that layer in the meantime. In theory any valid WFS could be used for testing if that still fails.

**Comment by JakobMiksch** [comment]
I used a different WMS for testing. Thanks. You can close this issue if you want.

**Comment by geographika** [comment]
Closed with #201
